### PR TITLE
Add functionality for sick people - aria and tabindex

### PR DESCRIPTION
**Pull request by glanttz**

### CODE REVIEW

File: src/components/prescriptions/ImportGroupComponent.vue [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <template>
     <section class="prescriptionContainer">
-        <h3 class="prescriptionContainer__header">
+        <h3
+            class="prescriptionContainer__header"
+            tabindex="0">
             Zaimportowane recepty
             <span class="numOfPres">{{
```
Comment (PixelSculptor) [inline src/components/prescriptions/ImportGroupComponent.vue:7]: did you check whether SR reads num of elemt of list? If not youhave to implement it in span markup

File: src/components/prescriptions/PrescriptionComponent.vue [hunk 2]
```diff
@@ -1,8 +1,18 @@
 <template>
     <li class="documentBox">
         <font-awesome-icon :icon="classOfIcon" />
-        <h5 class="documentBox__title">{{ name }}</h5>
-        <p class="documentBox__data">{{ date }}</p>
+        <h5
+            :aria-label="'Imię i nazwisko lekarza' + name"
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescriptionComponent.vue]: add ': '

File: src/components/prescriptions/PrescriptionComponent.vue [hunk 3]
```diff
@@ -1,8 +1,18 @@
...
+            class="documentBox__title"
+            tabindex="0">
+            {{ name }}
+        </h5>
+        <p
+            :aria-label="'Data wystawienia recepty' + date"
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescriptionComponent.vue]: add ': '

File: src/components/prescriptions/PrescriptionDetails.vue [hunk 4]
```diff
@@ -3,22 +3,42 @@
         <h3 class="detailsPres__header">Szczegóły recepty</h3>
         <form class="detailsPres__form detailsForm">
             <div class="inputBox">
-                <h5 class="detailsPres__title">Lekarz:</h5>
+                <h5
+                    :aria-label="'Imię i nazwisko lekarza' + names"
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescriptionDetails.vue]: add ': '

File: src/components/prescriptions/PrescriptionDetails.vue [hunk 5]
```diff
@@ -3,22 +3,42 @@
...
+                    tabindex="0">
+                    Lekarz:
+                </h5>
                 <p class="detailsPres__data">{{ names }}</p>
             </div>
 
             <div class="inputBox">
-                <h5 class="detailsPres__title">Nazwa leku:</h5>
+                <h5
+                    :aria-label="'Nazwa leku' + medicines"
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescriptionDetails.vue]: add ': '

File: src/components/prescriptions/PrescriptionDetails.vue [hunk 6]
```diff
@@ -3,22 +3,42 @@
...
+                    tabindex="0">
+                    Nazwa leku:
+                </h5>
                 <p class="detailsPres__data">{{ medicines }}</p>
             </div>
 
             <div class="inputBox">
-                <h5 class="detailsPres__title">Częstotliwość przyjmowania:</h5>
+                <h5
+                    :aria-label="'Częstotliwość przyjmowania' + frequencys"
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescriptionDetails.vue]: add ': '

File: src/components/prescriptions/PrescriptionDetails.vue [hunk 7]
```diff
@@ -3,22 +3,42 @@
...
+                    tabindex="0">
+                    Częstotliwość przyjmowania:
+                </h5>
                 <p class="detailsPres__data">{{ frequencys }}</p>
             </div>
 
             <div class="inputBox">
-                <h5 class="detailsPres__title">Data wystawienia recepty:</h5>
+                <h5
+                    :aria-label="'Data wystawienia recepty' + dates"
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescriptionDetails.vue]: add ': '

File: src/components/prescriptions/PrescriptionGroupComponent.vue [hunk 8]
```diff
@@ -1,6 +1,8 @@
 <template>
     <section class="prescriptionContainer">
-        <h3 class="prescriptionContainer__header">
+        <h3
+            class="prescriptionContainer__header"
+            tabindex="0">
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescriptionGroupComponent.vue:5]: same issue bout num of elements

File: src/components/prescriptions/PrescriptionImportComponent..vue [hunk 9]
```diff
@@ -2,7 +2,12 @@
     <li class="documentBox">
         <font-awesome-icon :icon="classOfIcon" />
         <h5 class="documentBox__title">{{ name }}</h5>
-        <p class="documentBox__data">{{ date }}</p>
+        <p
+            :aria-label="'Zaimportowano dnia' + date"
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescriptionImportComponent..vue]: add ': '

File: src/components/prescriptions/ImportGroupComponent.vue (hunk 1 above)
Comment (glanttz) [reply src/components/prescriptions/ImportGroupComponent.vue:7]: odczytywalo
przykladowo jak byly dwa to czytalo: zaimportowane recepty dwa

File: src/components/referrals/ImportReferralsComponent.vue [hunk 10]
```diff
@@ -25,9 +29,7 @@
 
             <div class="inputBox">
                 <label for="uploadTest">Dodaj wyniki:</label>
```
Comment (PixelSculptor) [inline src/components/referrals/ImportReferralsComponent.vue]: Is SR reads this?

File: src/components/referrals/ImportReferralsComponent.vue [hunk 11]
```diff
@@ -25,9 +29,7 @@
...
-                <div
-                    class="icon"
-                    tabindex="0">
+                <div class="icon">
                     <font-awesome-icon icon="fa-solid fa-file-import" />
```
Comment (PixelSculptor) [inline src/components/referrals/ImportReferralsComponent.vue]: add alt and aria label for icons everywhere where you use

File: src/components/referrals/ImportReferralsLabels.vue [hunk 12]
```diff
@@ -2,7 +2,12 @@
     <li class="documentBox">
         <font-awesome-icon :icon="classOfIcon" />
```
Comment (PixelSculptor) [inline src/components/referrals/ImportReferralsLabels.vue]: same

File: src/components/referrals/ImportReferralsLabels.vue [hunk 13]
```diff
@@ -2,7 +2,12 @@
...
         <h5 class="documentBox__title">{{ name }}</h5>
-        <p class="documentBox__data">{{ date }}</p>
+        <p
+            :aria-label="'Zaimportowano dnia' + date"
+            class="documentBox__data"
```
Comment (PixelSculptor) [inline src/components/referrals/ImportReferralsLabels.vue:10]: add ': '

File: src/components/referrals/ReferralsComponent.vue [hunk 14]
```diff
@@ -72,13 +76,15 @@
                 <button-component
                     :disabled="disableAddTest"
                     wide
+                    tabindex="0"
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsComponent.vue]: button is probably self-indexed. Please delete this and check if is indexed still

File: src/components/referrals/ReferralsComponent.vue [hunk 15]
```diff
@@ -72,13 +76,15 @@
...
                     >Zapisz skierowanie</button-component
                 >
                 <bouncing-balls-component :visible="isLoading" />
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsComponent.vue]: it is need to be added kind of description this animation

File: src/components/referrals/ReferralsComponent.vue [hunk 16]
```diff
@@ -72,13 +76,15 @@
...
                 <error-info :message="error" />
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsComponent.vue]: aria label and aria-live should be added

File: src/components/referrals/ReferralsComponent.vue [hunk 17]
```diff
@@ -72,13 +76,15 @@
...
                 <span
                     v-show="successFlag"
                     class="dataSend"
+                    tabindex="0"
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsComponent.vue:92]: aria-live probably should be added

File: src/components/referrals/ReferralsComponentLabel.vue [hunk 18]
```diff
@@ -1,8 +1,19 @@
 <template>
     <li class="documentBox">
         <font-awesome-icon :icon="classOfIcon" />
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsComponentLabel.vue]: alt and aria label

File: src/components/referrals/ReferralsComponentLabel.vue [hunk 19]
```diff
@@ -1,8 +1,19 @@
...
-        <h5 class="documentBox__title">{{ name }}</h5>
-        <p class="documentBox__data">{{ date }}</p>
+        <h5
+            :aria-label="'Imię i nazwisko lekarza' + name"
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsComponentLabel.vue]: add ': '

File: src/components/referrals/ReferralsComponentLabel.vue [hunk 20]
```diff
@@ -1,8 +1,19 @@
...
+            class="documentBox__title"
+            tabindex="0">
+            {{ name }}
+        </h5>
+
+        <p
+            :aria-label="'Data wystawienia skierowania' + date"
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsComponentLabel.vue]: I think that SR reads date without passing date to this aria atrr but i m not sure

File: src/components/referrals/ReferralsDetails.vue [hunk 21]
```diff
@@ -1,24 +1,46 @@
...
+        <h3
+            class="detailsReferrence__header"
+            tabindex="0">
+            Szczegóły skierowania
+        </h3>
         <form class="detailsReferrence__form detailsReferrence">
             <div class="inputBox">
-                <h5 class="detailsReferrence__title">Lekarz:</h5>
+                <h5
+                    :aria-label="'Imię i nazwisko lekarza' + names"
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsDetails.vue]: add ': '

File: src/components/referrals/ReferralsDetails.vue [hunk 22]
```diff
@@ -1,24 +1,46 @@
...
+                    class="detailsReferrence__title"
+                    tabindex="0">
+                    Lekarz:
+                </h5>
                 <p class="detailsReferrence__data">{{ names }}</p>
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsDetails.vue]: its not indexed and probably SR not read this during tabbing

File: src/components/referrals/ReferralsDetails.vue [hunk 23]
```diff
@@ -1,24 +1,46 @@
...
 
             <div class="inputBox">
-                <h5 class="detailsReferrence__title">Specjalizacja:</h5>
+                <h5
+                    :aria-label="'Specjalizacja lekarza' + specializations"
+                    class="detailsReferrence__title"
+                    tabindex="0">
+                    Specjalizacja:
+                </h5>
                 <p class="detailsReferrence__data">{{ specializations }}</p>
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsDetails.vue]: lack of index

File: src/components/referrals/ReferralsDetails.vue [hunk 24]
```diff
@@ -1,24 +1,46 @@
...
 
             <div class="inputBox">
-                <h5 class="detailsReferrence__title">Typ badania:</h5>
+                <h5
+                    :aria-label="'Typ badania' + typeOfTests"
+                    class="detailsReferrence__title"
+                    tabindex="0">
+                    Typ badania:
+                </h5>
                 <p class="detailsReferrence__data">{{ typeOfTests }}</p>
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsDetails.vue]: lack of index

File: src/components/referrals/ReferralsDetails.vue [hunk 25]
```diff
@@ -1,24 +1,46 @@
...
 
             <div class="inputBox">
-                <h5 class="detailsReferrence__title">
+                <h5
+                    :aria-label="'Data wystawienia skierowania' + dates"
+                    class="detailsReferrence__title"
+                    tabindex="0">
                     Data wystawienia skierowania:
                 </h5>
                 <p class="detailsReferrence__data">{{ dates }}</p>
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsDetails.vue]: lack of index

File: src/components/referrals/ReferralsGroupComponent.vue [hunk 26]
```diff
@@ -1,6 +1,8 @@
 <template>
     <section class="referralsContainer">
-        <h3 class="referralsContainer__header">
+        <h3
+            class="referralsContainer__header"
+            tabindex="0">
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsGroupComponent.vue:5]: not sure about reading num of items same as beginning of page

File: src/components/prescriptions/ImportGroupComponent.vue (hunk 1 above)
Comment (PixelSculptor) [reply src/components/prescriptions/ImportGroupComponent.vue:7]: ok so resolved

File: src/components/prescriptions/PrescriptionGroupComponent.vue (hunk 8 above)
Comment (glanttz) [reply src/components/prescriptions/PrescriptionGroupComponent.vue:5]: its counting properly

File: src/components/referrals/ReferralsDetails.vue (hunk 22 above)
Comment (glanttz) [reply src/components/referrals/ReferralsDetails.vue]: i delete it from h5 and add to p

File: src/components/referrals/ImportReferralsComponent.vue (hunk 10 above)
Comment (glanttz) [reply src/components/referrals/ImportReferralsComponent.vue]: yes, read properly

File: src/components/referrals/ReferralsComponentLabel.vue (hunk 20 above)
Comment (glanttz) [reply src/components/referrals/ReferralsComponentLabel.vue]: no, in SR read properly

File: src/components/referrals/ReferralsGroupComponent.vue (hunk 26 above)
Comment (glanttz) [reply src/components/referrals/ReferralsGroupComponent.vue:5]: its reding good bro 👯

File: src/components/prescriptions/PrescriptionAddComponent.vue [hunk 27]
```diff
@@ -64,10 +64,18 @@
...
-                <bouncing-balls-component :visible="isLoading" />
-                <error-info :message="error" />
+                <bouncing-balls-component
+                    role="presentation"
+                    alt="loader"
+                    aria-label="loader"
+                    :visible="isLoading" />
+                <error-info
+                    alt="error value"
+                    aria-label="error value"
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescriptionAddComponent.vue]:
```suggestion
                    :aria-label="error"
```

File: src/components/prescriptions/PrescriptionAddComponent.vue [hunk 28]
```diff
@@ -64,10 +64,18 @@
...
+                    :message="error" />
                 <span
                     v-show="successFlag"
+                    aria-live="prescription send"
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescriptionAddComponent.vue]:
```suggestion
                    aria-live="polite"
```

Comment (PixelSculptor) [reply src/components/prescriptions/PrescriptionAddComponent.vue]: aria-live is not for adding message. it is used to indicate in which moment SR should read. There are few params such as polite. you have to also add aria-label

File: src/components/prescriptions/PrescriptionAddComponent.vue [hunk 29]
```diff
@@ -176,19 +184,6 @@
                         @include text-button($font-weight-semiBold);
                     }
                 }
-
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescriptionAddComponent.vue:179]: why deleted

File: src/components/prescriptions/PrescriptionImport.vue [hunk 30]
```diff
@@ -75,10 +81,18 @@
...
-                <bouncing-balls-component :visible="isLoading" />
-                <error-info :message="error" />
+                <bouncing-balls-component
+                    role="presentation"
+                    alt="loader"
+                    aria-label="loader"
+                    :visible="isLoading" />
+                <error-info
+                    alt="error value"
+                    aria-label="error value"
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescriptionImport.vue]:
```suggestion
                    :aria-label="error"
```

Comment (PixelSculptor) [reply src/components/prescriptions/PrescriptionImport.vue]: also add aria-live

File: src/components/prescriptions/PrescriptionImport.vue [hunk 31]
```diff
@@ -75,10 +81,18 @@
...
+                    :message="error" />
                 <span
                     v-show="successFlag"
+                    aria-live="prescription import send"
```
Comment (PixelSculptor) [inline src/components/prescriptions/PrescriptionImport.vue]: wrong same as above. aria-live has predefined params. add polite

Comment (PixelSculptor) [reply src/components/prescriptions/PrescriptionImport.vue]:
```suggestion
                    aria-label="Recepta została przesłana pomyślnie"
```

File: src/components/referrals/ImportReferralsComponent.vue [hunk 32]
```diff
@@ -29,14 +29,22 @@
...
+                        icon="fa-solid fa-file-import" />
                 </div>
                 <input
                     id="uploadTest"
                     type="file"
                     @change="handleChange" />
-                <error-info :message="testFileError" />
+                <error-info
+                    alt="error file value"
+                    aria-label="error file value"
```
Comment (PixelSculptor) [inline src/components/referrals/ImportReferralsComponent.vue]:
```suggestion
                    :aria-label="error"
```

File: src/components/referrals/ImportReferralsComponent.vue [hunk 33]
```diff
@@ -45,10 +53,15 @@
...
-                <bouncing-balls-component :visible="isLoading" />
+                <bouncing-balls-component
+                    role="presentation"
+                    alt="loader"
+                    aria-label="loader"
+                    :visible="isLoading" />
                 <error-info :message="error" />
                 <span
                     v-show="successFlag"
+                    aria-live="import referral send"
```
Comment (PixelSculptor) [inline src/components/referrals/ImportReferralsComponent.vue]:
```suggestion
                    aria-label="Skierowanie zostało przesłane pomyślnie"
```

Comment (PixelSculptor) [reply src/components/referrals/ImportReferralsComponent.vue]: add aria live polite

File: src/components/referrals/ReferralsComponent.vue [hunk 34]
```diff
@@ -2,11 +2,7 @@
     <div
         class="referralsBox"
         @submit.prevent="addTest">
-        <h3
-            class="referralsBox__header"
-            tabindex="0">
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsComponent.vue]: why deleted? is SR reads without index during tabbing?

File: src/components/referrals/ReferralsComponent.vue [hunk 35]
```diff
@@ -76,14 +72,21 @@
...
-                <bouncing-balls-component :visible="isLoading" />
-                <error-info :message="error" />
+                <bouncing-balls-component
+                    role="presentation"
+                    alt="loader"
+                    aria-label="loader"
+                    :visible="isLoading" />
+                <error-info
+                    alt="error value"
+                    aria-label="error value"
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsComponent.vue]:
```suggestion
                    :aria-label="error"
```

Comment (PixelSculptor) [reply src/components/referrals/ReferralsComponent.vue]: for user info: 'error value' means nothing. It should be readable communicate

File: src/components/referrals/ReferralsComponent.vue [hunk 36]
```diff
@@ -76,14 +72,21 @@
...
+                    :message="error" />
                 <span
                     v-show="successFlag"
                     class="dataSend"
+                    aria-live="referral send"
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsComponent.vue]: same: aria live with params 
add also aria label about sending data

File: src/components/referrals/ReferralsComponent.vue [hunk 37]
```diff
@@ -190,18 +193,6 @@
                         @include text-button($font-weight-semiBold);
                     }
                 }
-                .icon {
```
Comment (PixelSculptor) [inline src/components/referrals/ReferralsComponent.vue:187]: why deleted

File: src/components/prescriptions/PrescriptionAddComponent.vue (hunk 29 above)
Comment (glanttz) [reply src/components/prescriptions/PrescriptionAddComponent.vue:179]: if it not used in form i think i can delete it.

File: src/components/referrals/ReferralsComponent.vue (hunk 37 above)
Comment (glanttz) [reply src/components/referrals/ReferralsComponent.vue:187]: as above

**Review by PixelSculptor — APPROVED** [review]

**Pull request merged**